### DISCUSSION TITLE
Per workspace bookmarks

### DISCUSSION THREAD

**Discussion by mr-cheffy** [zen-browser/desktop#885] [Ideas]
### What feature would you like to see?

I'm not sure if this is possible, but if you could have a different set of bookmarks per workspace that would be great.

For example, on workspace 1 you'd have Bookmark A, Bookmark B etc on the bookmark bar. On Workspace2, you wouldn't have Bookmarks A and B, but C, D etc etc

### What platform are you seeing the problem on?

Linux

 
 ~ _Created automatically from issue: https://github.com/zen-browser/desktop/issues/183_

**Comment by ChristinWhite** [comment]
Once tab groups are added, would this still be helpful? For my needs being able to nest unloaded tabs organized into folders would this type of bookmarks for me, essentially handling it the way Arc does, and they've completely eliminated bookmarks in favor of that. 

The exception however is that bookmarks do support tagging, which I still find useful.

**Reply by robotkutya** [reply]
Yes, very much so!

I want to use bookmarks, simply for "long term storage". Think future references kind of things, e.g. looking up a bread recipe that I saved 5 years ago.

I don't want to show my personal bookmarks during my work, where I have to share my screen and browser a lot. This is a very key feature for me that currently limits my use of the app in a professional context.

Love the work you all are doing, it's great, so refreshing to see some innovation in this space! Awesomesauce <3

**Comment by enoversum** [comment]
Adding my vote. I currently use two (separately synced) Brave Browser profiles for exactly that: Different settings, different logins, different bookmarks, as my work bookmarks are not interesting for my personal stuff, and vice versa. I didn't really get to like Arc's way of handling bookmarks, I'm afraid.

**Comment by Keltirion** [comment]
Also interested in this, i have 4 workspaces. Three for different clients i work for and fourth which is my personal. I need different links in different workspaces for quick access. Tab bar gets filled fast when i work with terraform for example and open 15 different resource docs in tabs. I ditched ARC cause of that. Hope it could be optional here.

**Comment by Shamatix** [comment]
Workspace bookmarks, history and the possibility for only pinned tabs to start up on new sessions, so if I have 10+ tabs open when I close the browser, I dont want these on next session, just my pinned tabs for my **workspace**

**Comment by hejchristiandev** [comment]
Would love to see this along with more Workspace separation in general (per workspace extensions for example)
I use each Workspace for completely different projects, each has a variety of different webpages (tools, documentation, etc) I'd like quick access too if needed. 

I am making do with folders for now but it'd be really nice to have a different set of bookmarks per space 👌

**Reply by Shamatix** [reply]
I made a post as well in regards to more separation, such as extensions, history etc as well :)

**Reply by robotkutya** [reply]
Firefox already has "containers"... I was wondering if we could have the same for workspaces, so each workspace by default is also a different container.

**Reply by hejchristiandev** [reply]
I do believe this is possible already in Zen with a bit of setup. 
You can right-click the workspace and set a default container, which would apply to every tab within. 

![CleanShot 2024-10-28 at 14 18 37](https://github.com/user-attachments/assets/7704c4d8-dbc3-472e-a7df-e2cf986d9616)

**Comment by estelle4565** [comment]
I would also like that. I have multiple different workspaces for different functions (different work workspaces + personal) and I would really like to be able to have a different bookmarks bar for each

**Comment by RavioEnby** [comment]
I thought it was good too when I am using ARC too.
However, if you have 10 folders in a workspace maybe for archive projects, the sidebar will now look a bit overwhelming.

Yet, ARC could create a whole new workspace from a folder and turn it back to folder at the initial workspace after all the editing.
It does work but I really hate the idea of using a workspace as a bookmark holder. It is 2 different things and ARC somehow put them together.

BUT!!!
switching folder and workspace is really brilliant idea
